bigchaindb_driver/driver.py

```diff
@@ -35,6 +36,8 @@ def __init__(self, *nodes, transport_class=Transport, headers=None):
         self._transport = transport_class(*self._nodes, headers=headers)
         self._transactions = TransactionsEndpoint(self)
         self._outputs = OutputsEndpoint(self)
+        self.api_version = api_version
+        self.api_prefix = '/api/{}'.format(api_version)
```

Comment:
let's just assume `'v'` will _always_ be, so we can pass a simple integer for the api version